src/core/AutomatableModel.cpp

```diff
@@ -713,6 +714,23 @@ float AutomatableModel::globalAutomationValueAt( const MidiTime& time )
 	}
 }
 
+float FloatModel::getRoundedValue() const
+{
+	return static_cast<float>( static_cast<int>( value() / step<float>() + 0.5 ) ) * step<float>();
```

Comment:
So this is causing a problem in the tripleoscillator coarse detune knobs: 
![image](https://cloud.githubusercontent.com/assets/6282045/25557998/ee14c8ec-2d1d-11e7-902d-4cc69898f3ae.png)
The negative values are off by one. 
And when you set it to zero
![image](https://cloud.githubusercontent.com/assets/6282045/25558001/06e342ea-2d1e-11e7-914e-45d98aaab905.png)
The actual value is -1 as you fixed this to use the true value in #3437 
![image](https://cloud.githubusercontent.com/assets/6282045/25558014/3711899a-2d1e-11e7-9d20-dcf186317dfb.png)